include/swift/Runtime/RuntimeFunctions.def

```diff
@@ -769,7 +769,7 @@ FUNCTION(GetFunctionMetadata, swift_getFunctionTypeMetadata, DefaultCC,
               TypeMetadataPtrTy->getPointerTo(0),
               Int32Ty->getPointerTo(0),
               TypeMetadataPtrTy),
-         ATTRS(NoUnwind, ReadNone))
+         ATTRS(NoUnwind))
```

Comment:
It can at least remain ReadOnly, can't it?

Reply:
Yes, I think so too. But this is the same state we have on master. So for swift-4.1-branch I don't want to diverge.

Reply:
https://github.com/apple/swift/blob/master/include/swift/Runtime/RuntimeFunctions.def#L749

Reply:
https://github.com/apple/swift/pull/14302

Reply:
Sounds good.